twilio/twiml/messaging_response.py

```diff
@@ -0,0 +1,128 @@
+from twilio.twiml import TwiML
+
+
+class MessagingResponse(TwiML):
+    """
+    Messaging TwiML Response
+    """
+    def __init__(self):
+        """
+        Create a new <Response>
+        """
+        super(MessagingResponse, self).__init__()
+        self.name = 'Response'
+
+    def message(self,
+                body,
+                to=None,
+                from_=None,
+                method=None,
+                action=None,
+                status_callback=None,
+                **kwargs):
```

Comment:
I like the other style where you keep them inline until the length limit then wrap to the next line. but not a big deal.